services/users.service.ts

```diff
@@ -15,6 +15,8 @@ mongoose.connection.on('close', () => console.log('User: close'))
 
 export default class UserService {
 
+    roles:string [] = ["Admin", "Manager", "User"]
```

Comment:
you could create an ENUM of roles and then by default assign User enum value to the role in registration, would be more professional